core/indigo-core/molecule/src/molecule_json_saver.cpp

```diff
@@ -1187,72 +1255,66 @@ void MoleculeJsonSaver::saveMonomerTemplate(TGroup& tg, JsonWriter& writer)
     writer.EndObject();
 }
 
+void MoleculeJsonSaver::saveSuperatomAttachmentPoints(Superatom& sa, JsonWriter& writer)
```

Comment:
this logic should not be part of json saver

Reply:
Fixed